### PR TITLE
Queues Live Pipeline + APIViews Feedback

### PR DESCRIPTION
**Pull request by tasherif-msft**
Remove `item` suffix from generated and client code.
Add live pipeline.

### CODE REVIEW

**Comment by tasherif-msft** [comment]
@benbp I'm trying to create a live pipeline for the `azqueue` package, is the addition of the arm template (test_resources.json) the only step required? please advise

File: sdk/storage/azqueue/service_client_test.go [hunk 1]
```diff
@@ -281,7 +281,7 @@ func (s *RecordedTestSuite) TestServiceListQueuesWithMetadata() {
 	for pager.More() {
 		resp, err := pager.NextPage(context.Background())
 		_require.Nil(err)
-		for _, queue := range resp.QueueItems {
+		for _, queue := range resp.QueuesList {
```
Comment (vibhansa-msft) [inline sdk/storage/azqueue/service_client_test.go:284]: QueueItems sound more intutive then QueueList (does not clearly tell what it means)

Comment (tasherif-msft) [reply sdk/storage/azqueue/service_client_test.go:284]: I'm ok with either or here, @jhendrixMSFT what's your input on this?

Comment (jhendrixMSFT) [reply sdk/storage/azqueue/service_client_test.go:284]: Is the suffix even necessary, can it just be `Queue`?

Comment (tasherif-msft) [reply sdk/storage/azqueue/service_client_test.go:284]: @jhendrixMSFT so `Queues`? I'm ok with that as well.

**Review by siminsavani-msft — APPROVED** [review]

**Pull request merged**